tests/tests_integration/test_api/test_workflows.py

```diff
@@ -119,28 +121,29 @@ def handle(client, data: dict):
         output["product"] = output["a"] * output["b"]
         return output
 
-    deployed = cognite_client.functions.create(name="Multiply", external_id=external_id, function_handle=handle)
-    return deployed
+    cognite_client.functions.create(name="Multiply", external_id=external_id, function_handle=handle)
+    pytest.skip("Function need to be redeployed, skipping tests that need it", allow_module_level=True)
```

Comment:
Is it worth adding to the "add" function as well?

Reply:
Forgot it, done now.